gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Mon Aug 09 17:29:31 IST 2021
```

Comment:
Same about Gradle issues, we will fix it in other PR.

Reply:
modified